backend/apps/sandbox/views.py

```
@@ -973,3 +973,60 @@ def my_attempts(self, request):
+
+
+class ADRScenarioViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    Viewset for ADR Scenarios.
+    """
+
+    queryset = (
+        ADRScenario.objects.all().prefetch_related("options").order_by("-created_at")
+    )
+    serializer_class = ADRScenarioSerializer
+    permission_classes = [permissions.AllowAny]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)